samples/v2/redis/HelloSequence.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+
+namespace VSSample
+{
+    public static class HelloSequence
+    {
+        [FunctionName("E1_HelloSequence")]
```

Comment:
I suggest we remove the "E1_" prefix from all of these. They were really only inteded for csx samples where I wanted file names to be ordered in the Azure portal. Since this is a fresh sample, that legacy probably shoudn't be carried forward.